nupic/encoders/scalar.py

```diff
@@ -30,6 +32,11 @@
 
 
 
+DEFAULT_RADIUS = 0
```

Comment:
Really odd that these are `0` instead of `None` but outside the scope of this change.